### PR TITLE
Added iOS installation/build instructions

### PR DESCRIPTION
**Pull request by Nexuist**
I wrote a step-by-step guide on how to build and configure a SwiftDiscord Xcode project so that it can run on iOS. In addition, I tried to describe common pitfalls during the process and how to avoid them.

Thanks for your help in making this happen, @nuclearace!

### CODE REVIEW

File: Installing SwiftDiscord for iOS.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Installing SwiftDiscord for iOS
+
+**Please note that you should follow the instructions in this document rather than in the main README if you intend to build for iOS.**
+
+
+## Preamble
+
+Hi there, I'm Andi. I wanted to build an iOS app that used the Discord API and SwiftDiscord fit the bill for my usecase. Unfortunately there were no official instructions at the time so I decided to try to do it  myself and document the whole process so the rest of you don't have to struggle. I'd like to thank the lead developer (nuclearace) for providing guidance throughout the whole journey; without whom I would have undoubtedly never been able to get this working.
```
Comment (nuclearace) [inline Installing SwiftDiscord for iOS.md]: I appreciate the preamble, but I don't think this is really needed to install iOS. You could maybe shorten it and move it to the bottom in a `## Thanks` perhaps.

File: Installing SwiftDiscord for iOS.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+* SwiftDiscord **4**
+* Xcode **8.3.3**
+* iOS SDK **10.3**
+* Homebrew
+
+I have absolutely no idea if this will continue to work with future or previous versions of SwiftDiscord/Xcode/iOS. Your mileage may vary.
+
+## Instructions
+
+You may be tempted to skip some steps or to not follow them in order. If you'd like to get this working at any point within the next few hours, I highly recommend you **do not do that**. If you do somehow get this to work without completing a certain step, please let me know so I can update the instructions.
```
Comment (nuclearace) [inline Installing SwiftDiscord for iOS.md]: Instead of the subjective `I` it's best to use something neutral, like `it`.

**Review by nuclearace — CHANGES_REQUESTED** [review]
This looks good overall. Would you move this to `UsageDocs` so that it gets built with the rest of the other docs?

**Comment by nuclearace** [comment]
Also the PR should be opened against development.

**Comment by Nexuist** [comment]
Let me know if anything else needs to be changed. Thanks again :)

**Review by nuclearace — APPROVED** [review]

**Pull request merged**